src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -440,7 +440,24 @@ internal bool RuntimeSupportsStaticAbstractMembersInInterfaces
             get => RuntimeSupportsFeature(SpecialMember.System_Runtime_CompilerServices_RuntimeFeature__VirtualStaticsInInterfaces);
         }
 
-        private bool RuntimeSupportsFeature(SpecialMember feature)
+        /// <summary>
+        /// Whether the target runtime supports numeric IntPtr types.
+        /// This test hook should be removed once TargetFramework.Net70 is added.
+        /// Tracked by https://github.com/dotnet/roslyn/issues/61235
+        /// </summary>
+        internal virtual bool RuntimeSupportsNumericIntPtr
```

Comment:
>virtual [](http://example.com/codeflow?start=17&length=7)

Where is this base implementation used? It looks like the only `AssemblySymbol` implementation that does not override this is `RetargetingAssemblySymbol`. If so, consider making this `abstract` and adding an appropriate override for that case.

Reply:
There's also `MissingAssemblySymbol`.
FWIW, I'm following the model from original [DIM PR](https://github.com/dotnet/roslyn/pull/17927), trying to stick to an established pattern as much as possible (since we're going to have to do this regularly).